### PR TITLE
Add Facilitating Mesh Workshops event in Dec 2018

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: _posts/2018-12-17-facilitating-mesh-workshops.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+slug: facilitating-mesh-workshops
+text: 'Learn how to facilitate the series'
+location: Free Geek Toronto, 180 Sudbury St
+locationLink: https://osm.org/go/ZX6BI0AGy?node=4042949045
+date: 2018-12-17
+startTime: '18:30'
+endTime: '21:00'
+---
+
+This is a session to prepare future facilitators to organize a _Building the Peer-to-Peer Internet_ workshop, in part or in full, and variations of it. Participants are assumed familiarity with social motivations and tensions around distributed systems, basic networking concepts, and are comfortable working in command line environments.
```
Comment (Shrinks99) [inline _posts/2018-12-17-facilitating-mesh-workshops.md]: "Participants are assumed familiarity with" should probably be changed to "Participants should be familiar with...".

**Review by Shrinks99 — COMMENTED** [review]
Minor grammatical fix, other than that this looks good to me

File: _posts/2018-12-17-facilitating-mesh-workshops.md (hunk 1 above)
Comment (benhylau) [reply _posts/2018-12-17-facilitating-mesh-workshops.md]: Thanks

**Review by dasanchez — APPROVED** [review]
Thanks for putting this together @benhylau!

**Review by garrying — APPROVED** [review]

**Pull request merged**